docs/tutorial/getting-started/life-cycle/index.md

```
@@ -56,12 +56,14 @@ new Elysia()
 			console.log('This is executed before handler')
 
 			if(Math.random() <= 0.5)
-				return status(418)
+				return status(418) // or status("I'm a teapot")
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)